x-pack/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/installation_and_upgrade.md

```diff
@@ -815,6 +821,122 @@ And the Investigation Guide tab should NOT be displayed
 
 TODO: add scenarios https://github.com/elastic/kibana/issues/166215
 
+### Rule upgrade workflow: simple diff algorithm
+
+#### **Scenario: Rule field doesn't have an update and has no custom value**
+
+**Automation** 1 integration test
```

Comment:
- Shouldn't we have 2 tests since we're covering two fields in the scenario?
- Let's mention that we will use mock rules instead of the real package (see other scenarios).
- Let's mention unit test coverage as well.

```suggestion
**Automation**: 2 integration tests with mock rules + a set of unit tests for the algorithm.
```